docs/dashboard-spec.md

```diff
@@ -14,6 +14,31 @@ packs/<pack_name>/dashboards/apache.json
 
 The schema adheres to the `DashboardInput` used in the [dashboardCreate](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/dashboards-api-migration-insights-api-nerdgraph/#dashboard-create) mutation in NerdGraph.
 
+
+```yaml
+####################################################################################
+# Metadata
+####################################################################################
+
+# the name of the Dashboard
+name: string, required
+
+# A short description for this Dashboard
+description: string, required
+
+# List of screenshot URLs related to this Dashboard
+type: list, required
```

Comment:
This field is currently not required in the json schema

Reply:
It shows are requires in nerdgraph though...


<img width="361" alt="Screen Shot 2021-08-19 at 8 29 08 AM" src="https://user-images.githubusercontent.com/1395158/130086835-6d47fc03-b414-42d9-a2a8-ec18e9ce292c.png">

Reply:
hol up, this means the field is

`String!` means that the field is non-nullable, meaning that the GraphQL service promises to always give you a value when you query this field. In the type language, we'll represent those with an exclamation mark.

I'll fix it. I misunderstood the `!`

Reply:
I'm confirming what this actually means as it relates to this field.

Reply:
@rudouglas did confirm this `string!` does mean required..